<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="nh.ui.antomation.test.cases.Case2" tests="1" time="1.109" ignored="0" failures="1" timestamp="16 Nov 2017 15:23:50 GMT" skipped="0" hostname="MS-20161002OWKF" errors="0">
  <testcase name="baidu_search" time="1.109" classname="nh.ui.antomation.test.cases.Case2">
    <failure type="java.lang.AssertionError" message="没有找到测试对象,对象名称[bdSearchInput]!">
      <![CDATA[java.lang.AssertionError: 没有找到测试对象,对象名称[bdSearchInput]!
	at org.testng.Assert.fail(Assert.java:94)
	at nh.ui.automation.tools.common.ByType.getByType(ByType.java:56)
	at nh.ui.automation.tools.common.Action.objInfo(Action.java:97)
	at nh.ui.automation.tools.common.ElementCommon.getElement(ElementCommon.java:36)
	at nh.ui.automation.tools.common.Action.sendkey(Action.java:126)
	at nh.ui.antomation.test.cases.Case2.baidu_search(Case2.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- baidu_search -->
</testsuite> <!-- nh.ui.antomation.test.cases.Case2 -->
